terminal/noninteractive.go

```diff
@@ -212,6 +215,51 @@ func (ui *nonInteractiveUI) Error(msg string) {
 func (ui *nonInteractiveUI) ErrorWithContext(err error, sub string, ctx ...string) {
 	ui.Error(helper.Title(sub))
 	ui.Error("  Error: " + err.Error())
+
+	// Selectively promote Details and Suggestion from the context.
+	var extractItem = func(ctx []string, key string) ([]string, string, bool) {
+		for i, v := range ctx {
+			if strings.HasPrefix(v, key) {
+				outStr := v
+				outCtx := slices.Delete(ctx, i, i+1)
+				return outCtx, outStr, true
+			}
+		}
+		return ctx, "", false
+	}
+	var promote = func(key string) {
+		if oc, item, found := extractItem(ctx, key); found {
+			ctx = oc
+			if key == "" {
+				return
+			}
+
+			key, rest, found := strings.Cut(item, ": ")
+
+			if !found {
+				wrapped := wordwrap.WrapString(rest, 78)
```

Comment:
https://pkg.go.dev/strings#Cut has returned `""` for `rest` if we're in this branch. Should this be?:

```suggestion
				wrapped := wordwrap.WrapString(key, 78)
```